src/frontend/Komponenter/Behandling/Modal/HenleggModal.tsx

```diff
@@ -85,8 +117,20 @@ export const HenleggModal: FC<{ behandling: Behandling }> = ({ behandling }) =>
     const lukkModal = () => {
         settFeilmelding('');
         settVisHenleggModal(false);
+        settHenlagtårsak(undefined);
     };
 
+    const tilknyttetFullmakt = fullmakt.some(
+        (f) => f.gyldigTilOgMed === null || erEtterDagensDato(f.gyldigTilOgMed)
```

Comment:
```suggestion
        (fullmakt) => fullmakt.gyldigTilOgMed === null || erEtterDagensDato(fullmakt.gyldigTilOgMed)
```

Dette er lettere å lese

Reply:
Da må nok variabelen med alle fullmakter renames til `fullmakter` i flertall for å unngå shadowing